### PR TITLE
feat: Add prefinance projects tab to projects list

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: #2277

Since #2288 (prefinance project card) is not merged yet and I needed the changes from there, I've based my work on this existing work. 

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

https://deploy-preview-2291--regen-marketplace.netlify.app/projects/1

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-marketplace/src/pages/Projects/AllProjects/AllProjects.CommunityFilter.tsx [hunk 1]
```diff
@@ -8,9 +8,9 @@ import InfoTooltipWithIcon from 'web-components/src/components/tooltip/InfoToolt
 import { useCommunityProjectsAtom } from 'lib/atoms/projects.atoms';
```
Comment (blushi) [inline web-marketplace/src/pages/Projects/AllProjects/AllProjects.CommunityFilter.tsx:8]: a good part of the changes here is about moving the existing Projects component to Projects/AllProjects so eventually we have the following structure:
- Projects
  - AllProjects tab content
  - PrefinanceProjects tab content

**Comment by blushi** [comment]
@erikalogie you can test this on https://deploy-preview-2291--regen-marketplace.netlify.app/projects/1

**Comment by erikalogie** [comment]
@blushi this looks great! Can you just make the prefinance icon smaller and the font size of the tabs smaller on mobile? https://monosnap.com/file/mkGw9F1CnQamhUDrOKbM4FcoQ0drEr.  Like in this comp https://www.figma.com/file/EPSVuGURedvRfO6LxUAQZs/Pre-financing?type=design&node-id=663%3A39817&mode=design&t=iGDAkNYTdOS198Jo-1

Also the space between the tabs on the desktop should be bigger: https://monosnap.com/file/MbA9pQ1R1za0Xf3ahy5MZbBMZlSMGz

If it is possible to make the bracketed project amounts Mulish Medium instead of black, I also think that looks better but I know it is a one-off that I don't have in the styles.

**Comment by blushi** [comment]
> @blushi this looks great! Can you just make the prefinance icon smaller and the font size of the tabs smaller on mobile? https://monosnap.com/file/mkGw9F1CnQamhUDrOKbM4FcoQ0drEr. Like in this comp https://www.figma.com/file/EPSVuGURedvRfO6LxUAQZs/Pre-financing?type=design&node-id=663%3A39817&mode=design&t=iGDAkNYTdOS198Jo-1
> 
> Also the space between the tabs on the desktop should be bigger: https://monosnap.com/file/MbA9pQ1R1za0Xf3ahy5MZbBMZlSMGz
> 
> If it is possible to make the bracketed project amounts Mulish Medium instead of black, I also think that looks better but I know it is a one-off that I don't have in the styles.

I've addressed those.

**Comment by erikalogie** [comment]
Looks great!!

**Review by aaronc — APPROVED** [review]

**Pull request merged**